docker-compose.yml

```diff
@@ -1,21 +1,56 @@
 # -*- coding: utf-8 -*-
 
+elasticsearch:
+  image: "elasticsearch:5"
+  ports:
+    - "9200:9200"
```

Comment:
When I try this using docker-machine for OSX, the image complains that it can't start.  Will look into it...